### PR TITLE
Comments and cleanup

### PR DESCRIPTION
**Pull request by AlexT2052**
Much comments and cleanup, got rid of almost all the warnings, removed some commented out code or things that we would never use again and would just be even more confusing for beginners looking through this for the first time. Also did comments and cleanup btw

### CODE REVIEW

File: src/main/java/frc/robot/commands/drive/VisionDriveCommand.java [hunk 1]
```diff
@@ -6,16 +6,28 @@
...
 
 public class VisionDriveCommand extends DefaultDriveCommand {
     private final VisionSubsystem vision;
-    private final TargetingSubsystem targeting;
+    // private final TargetingSubsystem targeting;
     private double visionRotation;
 
+    /**
+     * Command for making the robot aim at the hub and track it while driving if desired.
+     * Uses relatively arbitrary values but it worked and did so better than what we coulsd get with
```
Comment (LexiBirken) [inline src/main/java/frc/robot/commands/drive/VisionDriveCommand.java:20]: replace "coulsd" with could. Or don't, I don't know what our comment spelling standards are.

File: src/main/java/frc/robot/auto/AutoBase.java [hunk 2]
```diff
@@ -74,6 +73,10 @@ public class AutoBase extends SequentialCommandGroup {
      * Base class that all Auto classes inherited from. Contains easy command instanciation methods,
      * Pose2d and Rotation2d making methods, trajectory generation methods, and most importantly 
      * SwerveControllerCommand generation.
+     * 
+     * Refer to this slideshow for more information on this class and autos!
```
Comment (LexiBirken) [inline src/main/java/frc/robot/auto/AutoBase.java:77]: Alex, change this link to be "view only". Currently makes slides editable.

**Review by LexiBirken — COMMENTED** [review]
Looks great Alex! Left a few minor comments, on last couple commits. Love the detail and explanation of our code this year, makes it a lot more readable! Took a look at the earlier commits too ofc but didn't see anything that struck me as hazardous or anything. Great work!

**Review by generic-steve — APPROVED** [review]
you needed someone to click a button?

**Pull request merged**